src/Maker/MakeFunctionalTest.php

```diff
@@ -82,5 +84,11 @@ public function configureDependencies(DependencyBuilder $dependencies)
             true,
             true
         );
+        $dependencies->addClassDependency(
```

Comment:
And, I realized that we need a small tweak in `DependencyBuilder::calculateMissingDependencies()` - it should now also check for `trait_exists()` in there. Currently, it does not, so it would ALWAYS think this dependency wasn't installed.

Reply:
[Fixed there !](https://github.com/symfony/maker-bundle/pull/426)